parquet-column/src/main/java/org/apache/parquet/schema/GroupType.java

```diff
@@ -88,6 +88,7 @@ public GroupType(Repetition repetition, String name, OriginalType originalType,
    * @param fields the contained fields
    * @param id the id of the field
    */
+  @Deprecated
```

Comment:
I think, it is enough to deprecate public API. If a method is not public we can freely modify/remove it.